src/seedu/addressbook/parser/Parser.java

```diff
@@ -1,28 +1,14 @@
 package seedu.addressbook.parser;
 
-import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.addressbook.common.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+import seedu.addressbook.commands.*;
```

Comment:
Don't use wildcard imports - it is a coding standard violation. You may want to check if IntelliJ automatically doing this for you, and if so, reconfigure it so that it doesn't.